lib/internal/modules/cjs/loader.js

```diff
@@ -219,6 +220,10 @@ ObjectDefineProperty(Module, 'wrapper', {
   }
 });
 
+ObjectDefineProperty(Module.prototype, 'isPreloading', {
+  get() { return isPreloading; }
+});
```

Comment:
It might be a good idea to also backport `NativeModule.prototype.isPreloading`: <https://github.com/nodejs/node/blob/e2793049542b50bcbb07148cd17ed7c517faa467/lib/internal/modules/cjs/loader.js#L236-L238>